apps/sending-func/package.json

```diff
@@ -2,34 +2,37 @@
   "name": "sending-func",
   "version": "1.10.14",
   "license": "UNLICENSED",
+  "type": "commonjs",
   "scripts": {
     "build": "yarn openapi:bundle && yarn generate:definitions && yarn generate:backend-notification-hub && tsc",
     "watch": "tsc --w",
     "test": "vitest run",
     "test:coverage": "vitest run --coverage",
-    "lint": "eslint --fix .",
-    "lint:check": "eslint .",
+    "lint": "eslint --fix src",
+    "lint:check": "eslint src",
+    "start": "func start",
     "openapi:bundle": "yarn node copy-commons-definitions.js && yarn openapi:bundle:internal && yarn openapi:bundle:external",
     "openapi:bundle:internal": "bundle-api-spec -i openapi/index_internal.yaml.template -o openapi/index.yaml -V $npm_package_version",
     "openapi:bundle:external": "bundle-api-spec -i openapi/index_external.yaml.template -o openapi/index_external.yaml -V $npm_package_version",
-    "generate:definitions": "rimraf ./generated/definitions && shx mkdir -p ./generated/definitions && gen-api-models --api-spec ./openapi/index.yaml --no-strict --out-dir ./generated/definitions",
-    "generate:backend-notification-hub": "rimraf ./generated/notifications && shx mkdir -p generated/notifications && gen-api-models --strict 0 --api-spec https://raw.githubusercontent.com/pagopa/io-backend/v8.10.1/notification_queue_messages.yaml --out-dir generated/notifications",
+    "generate:definitions": "rimraf ./src/generated/definitions && shx mkdir -p ./src/generated/definitions && gen-api-models --api-spec ./openapi/index.yaml --no-strict --out-dir ./src/generated/definitions",
+    "generate:backend-notification-hub": "rimraf ./src/generated/notifications && shx mkdir -p ./src/generated/notifications && gen-api-models --strict 0 --api-spec https://raw.githubusercontent.com/pagopa/io-backend/v8.10.1/notification_queue_messages.yaml --out-dir ./src/generated/notifications",
     "version": "yarn bump:openapi:internal && yarn bump:openapi:external",
     "bump:openapi:internal": "bundle-api-spec -i openapi/index.yaml -o openapi/index.yaml -V $npm_package_version && git add openapi/index.yaml",
-    "bump:openapi:external": "bundle-api-spec -i openapi/index_external.yaml -o openapi/index_external.yaml -V $npm_package_version && git add openapi/index_external.yaml"
+    "bump:openapi:external": "bundle-api-spec -i openapi/index_external.yaml -o openapi/index_external.yaml -V $npm_package_version && git add openapi/index_external.yaml",
+    "format": "prettier --write .",
+    "format:check": "prettier --check .",
+    "typecheck": "tsc --noEmit"
   },
   "devDependencies": {
     "@azure/functions": "^3.2.0",
     "@pagopa/eslint-config": "^4.0.3",
     "@pagopa/openapi-codegen-ts": "^14.0.0",
     "@types/express": "^4.17.11",
-    "@types/html-to-text": "^1.4.31",
     "@types/node": "^20.17.30",
-    "@types/nodemailer": "^4.6.8",
+    "@types/node-fetch": "^2.5.7",
```

Comment:
In future we could remove this dependency in order to use the standard fetch

Reply:
Yes, we will remove `node-fetch` as part of a future monorepo-wide update but that’s out of scope for now. This refactor only impacted dev dependencies and type-level source code.